docs/source-pytorch/conf.py

```diff
@@ -49,8 +49,24 @@
 # -- Project documents -------------------------------------------------------
 if _SHOULD_COPY_NOTEBOOKS:
     AssistantCLI.copy_notebooks(
-        PATH_RAW_NB, PATH_HERE, "notebooks", patterns=[".", "course_UvA-DL", "lightning_examples"]
+        PATH_RAW_NB,
+        PATH_HERE,
+        "notebooks",
+        patterns=[".", "course_UvA-DL", "lightning_examples"],
     )
+    # TODO: Complete converting the missing items and add them back
+    ignore = [
+        "course_UvA-DL/13-contrastive-learning",
+        "lightning_examples/augmentation_kornia",
+        "lightning_examples/finetuning-scheduler",
+        "lightning_examples/reinforce-learning-DQN",
+        "lightning_examples/text-transformers",
+        "lightning_examples/warp-drive",
+    ]
+    for file in ignore:
+        file = os.path.join(PATH_HERE, "notebooks", file)
+        if os.path.exists(file):
+            os.remove(file)
```

Comment:
Is this going to create undesired local changes when creating the docs?

Reply:
Nope, the notebooks folder referenced here was always generated. It is basically the same type of artifact as the build folder for the docs. whatever we add or remove here won't be seen by the developer